src/components/Authentication/Signup.jsx

```diff
@@ -24,9 +25,19 @@ const Signup = ({ isAdmin }) => {
   // eslint-disable-next-line no-unused-vars
   const [error, setError] = useState('');
   const [loading, setLoading] = useState(false);
-
+  const toast = useToast();
   const navigate = useNavigate();
 
+  useEffect(() => {
+    toast({
+      title: 'Login Failed',
+      description: error,
+      status: 'error',
+      duration: 3000,
+      isClosable: true,
+    });
+  }, [error]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'toast'. Either include it or remove the dependency array.